scripts/check-for-header.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+FILES=$(find $PWD -type f \( -iname '*.go' ! -iname "*.pb.go" \) )
+for i in $FILES
+do
+  if ! grep -q 'Copyright' "$i"
+  then
+    echo " missing copyrioght header for $i"
```

Comment:
typo. also is it worth doing something like

```
if ! grep -q 'Copyright' "$(head -n 3 "$i")"
```

so we only check the first few lines?

Reply:
oh, i screwed up the review, but i still care about this comment